config/initializers/startup_jobs.rb

```diff
@@ -1,5 +1,5 @@
 # Be sure to restart your server when you modify this file.
 
 Rails.application.config.after_initialize do
-  ClusterStatusJob.perform_now(1)
+  ClusterStatusJob.perform_now('owens')
```

Comment:
Thanks, and sorry about all that. Do you want to add `whenever` now or in separate pull request?

Reply:
I can try adding whenever in this pr.

Reply:
cool.